components/date-picker/__docs__/storybook-stories.jsx

```diff
@@ -12,13 +12,26 @@ import CustomInput from '../__examples__/custom-input';
 import SnaphotDefault from '../__examples__/snapshot-default';
 import WeekdayPicker from '../__examples__/weekday-picker';
 
+// eslint-disable-next-line camelcase
+import UNSAFE_DirectionSettings from '~/components/utilities/UNSAFE_direction';
+
+const makeRtl = (component) => (
```

Comment:
I don't like how we are repeating makeRtl method. I will have another PR coming up for color picker. If no one objects I will move this to the direction/index.js and reuse.

@andyouyang22 @minevskiy

Reply:
I'm fine with extracting it into a test helper, but not sure it should be in `src`

Reply:
Yeah, I'd recommend something in `/test` folder. There are other helper files in there. `rtl-helpers.js.jsx` is good.